internal/pkg/scorecard/scorecard.go

```diff
@@ -321,42 +377,51 @@ func ScorecardTests(cmd *cobra.Command, args []string) error {
 		return err
 	}
 	cmd.SilenceUsage = true
-	suites, err := runTests()
+	pluginOutputs, err := runTests()
 	if err != nil {
 		return err
 	}
 	totalScore := 0.0
 	// Update the state for the tests
-	for _, suite := range suites {
-		for idx, res := range suite.TestResults {
-			suite.TestResults[idx] = UpdateState(res)
+	for _, suite := range pluginOutputs {
+		for idx, res := range suite.Results {
+			suite.Results[idx] = UpdateSuiteStates(res)
 		}
 	}
 	if viper.GetString(OutputFormatOpt) == "human-readable" {
```

Comment:
While it's not in this PR but can we make the output format flag values(`human-readable` and `json` ) into constants?
Right now they're appearing in several places as string literals.

Should be a small change, or you can make a separate PR for it.